src/typescript/frontend/src/components/pages/emojicoin/components/trade-emojicoin/SwapButton.tsx

```diff
@@ -52,13 +52,21 @@ export const SwapButton = ({
         (e) => e.type === STRUCT_STRINGS.LotteryWinnerEvent
       );
       if (rewardsEvent) {
-        controls.start("celebration");
-        toast.success(<CongratulationsToast transactionHash={res.response.hash} />, {
-          pauseOnFocusLoss: false,
-          autoClose: 7000,
-          position: "top-center",
-          closeOnClick: false,
-        });
+        toast.dark(
+          <>
+            <RewardsAnimation controls={controls} />
+            <CongratulationsToast transactionHash={res.response.hash} />
+          </>,
+          {
+            pauseOnFocusLoss: false,
+            autoClose: 7000,
```

Comment:
Just a reminder that we were talking about increasing this to about double, so maybe 15000 here

Reply:
Thank you. I would have forgotten if you did not remind me :sweat_smile: